query/stdlib/testing/end_to_end_test.go

```diff
@@ -0,0 +1,364 @@
+package testing_test
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	nethttp "net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/lang"
+	"github.com/influxdata/flux/parser"
+	"github.com/influxdata/flux/stdlib"
+
+	platform "github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/bolt"
+	"github.com/influxdata/influxdb/cmd/influxd/launcher"
+	"github.com/influxdata/influxdb/http"
+	"github.com/influxdata/influxdb/query"
+
+	_ "github.com/influxdata/flux/stdlib"           // Import the built-in functions
+	_ "github.com/influxdata/influxdb/query/stdlib" // Import the stdlib
+)
+
+// Default context.
+var ctx = context.Background()
+
+type AssertionError interface {
```

Comment:
This is likely not needed, also it doesn't work on this project since the error is re-encoded via the CSV encoder, and the type is lost.

Reply:
Should we remove this now?